src/internal/m365/collection/drive/collections_tree_test.go

```diff
@@ -247,8 +247,12 @@ type collectionAssertions map[string]collectionAssertion
 // 	}
 // }
 
-func rootAnd(items ...models.DriveItemable) []models.DriveItemable {
-	return append([]models.DriveItemable{driveItem(rootID, rootName, parent(0), "", isFolder)}, items...)
+func pageItems(items ...models.DriveItemable) []models.DriveItemable {
+	return append([]models.DriveItemable{driveRootItem()}, items...)
+}
+
+func drivePageItems(driveID any, items ...models.DriveItemable) []models.DriveItemable {
+	return append([]models.DriveItemable{driveRootItem()}, items...)
 }
```

Comment:
since the root item isn't actually populating the drive ID in the parent ref portion of the item (even though graph seems to) this function is exactly the same as `pageItems`. Could you please either switch `driveRootItem` to fill out the drive ID or just remove this function completely?

Reply:
Will remove.  I think I removed it in one of the later cleanups already, but might as well make the change here.